Source/Dafny/DafnyOptions.cs

```diff
@@ -27,6 +27,7 @@ public DafnyOptions()
       Prune = true;
       NormalizeNames = true;
       EmitDebugInformation = false;
+      Compiler = new PlaceholderCompiler();
```

Comment:
Do we really need this? It seems like we need something like a `DafnyOptionsBuilder` that forces you to create properly initialised DafnyOptions, meaning `ApplyDefaultOptions`, which maybe should be called `ReadyOptionsForUse`, is called at the end.

Reply:
I think we don't.  Let's just revert to the default of having C# out of the box and move on ^^